client/subcomponents/News/News.jsx

```diff
@@ -5,13 +5,33 @@ export default function News({ news }) {
   const createTime = moment(news.createdOn, 'DD/MM/YYYY').fromNow()
 
   return (
-    <article>
-      <h2>{news.title}</h2>
-      <h2>
-        By {news.firstName} {news.lastName}:
-      </h2>
-      <p>{news.content}</p>
-      <p>{createTime}</p>
-    </article>
+    <div className="relative rounded-3xl border-3 shadow-xl">
```

Comment:
don't forget to apply mobile styling to this component! looks like padding/margin adjustments and column numbers should do the trick

Reply:
Thanks, added these as well, yeah you're right padding does the work! Every time.